sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -938,6 +964,139 @@ The output of the `sem delete notification documents` command is as follows:
+
+Note that `sem rebuild pipeline` will perform a partial rebuild of the pipeline
+as **only the blocks that failed** will be rerun. Additionally, there is no way
+to see that pipeline running in the Semaphore web interface.
+
+#### Rebuilding a pipeline example
+
+Rebuilding the pipeline with a Pipeline ID of
+`b83bce61-fdb8-4ace-b8bc-18be471aa96e` will return the next output:
+
+    $ sem rebuild pipeline b83bce61-fdb8-4ace-b8bc-18be471aa96e
+    {"pipeline_id":"a1e45038-a689-460b-b3bb-ba1605104c08","message":""}
+
+## Working with workflows
+
+### Listing workflows
+
+The `sem get workflows` command returns the list of workflows for a Semaphore
+2.0 project. If you are inside the directory of a Semaphore project, then you
+will not need to provide `sem` the name of the project as this will be
+discovered automatically. However, if there is a problem with the format of the
+GitHub URL of your repository or if you are outside the directory of a GitHub
+repository you can use the `-p` flag to declare the Semaphore project that
+interests you.
+
+So the `sem get workflows` has the following two formats:
+
+    sem get workflows
+    sem get workflows -p [Project Name]
+
+### Listing workflows examples
+
+Finding the last 30 workflows for the S1 Semaphore project is as simple as
+executing the following commands:
+
+    sem get workflows -p S1
+
+If the project name does not exist, you will get an error message.
+
+If you are inside the root directory of the S1 Semaphore project, then you can
+also execute the next command to get the same output:
+
+    sem get workflows
+
+### Describing a workflow
+
+Each workflow has its own unique Workflow ID. Using that unique Workflow ID you
+can find more information about that particular workflow using the next command:
+
+    sem get workflow [WORKFLOW ID]
+
+The output of the previous command is a list of pipelines that includes
+promoted and auto promoted pipelines.
+
+#### Describing a workflow example
+
+Finding more information about the Workflow with ID
+`5bca6294-29d5-4fd3-891b-8ac3179ba196` is as easy as follows:
+
+    $ sem get workflow 5bca6294-29d5-4fd3-891b-8ac3179ba196
+    Label: master
+    
+    PIPELINE ID                            PIPELINE NAME            CREATION TIME         STATE
+    9d5f9986-2694-43b9-bc85-6fee47440ba7   Pipeline 1 - p1.yml      2018-10-03 09:37:59   DONE
+    55e94adb-7aa8-4b1e-b60d-d9a1cb0a443a   Testing Auto Promoting   2018-10-03 09:31:35   DONE
+
+Please note that you should have the required permissions in order to find
+information about a Workflow.
+
+### Rebuilding a workflow
+
+You can rebuild an existing workflow using the following command:
+
+    sem rebuild workflow [WORKFLOW ID]
+
+Note that `sem rebuild workflow` will perform a full rebuild of the specified
+workflow, which means that all blocks of the pipelines of the workflow will
+rerun. Additionally, the rebuild of the workflow will be visible in Semaphore
```

Comment:
> which means that all blocks of the pipelines of the workflow will rerun.

This is not true strictly speaking, since in the "original" workflow there might be some manually promoted pipelines and those would not be rescheduled.

I'd go with something like: 
"Note that `sem rebuild workflow` will perform a full rebuild of the specified
 workflow. New workflow will be created as if new commit was pushed to GitHub."

Reply:
Thanks.